src/components/ValidationCanvas.js

```diff
@@ -0,0 +1,35 @@
+import ManifestoCanvas from '../lib/ManifestoCanvas';
+
```

Comment:
I'd be fine w/ moving thing directly into `ManifestoCanvas` at some point if that makes sense.

Reply:
I thought, as ManifestoCanvas is kind of external code and may be modified and therefore needs to be replaced somedays it is easier to maintain, when inheriting from `ManifestoCanvas`. What do you think?

Reply:
`ManifestoCanvas` is an internal class that is just providing some additional convenience methods on Manifesto's Canvas objects. I guess the naming is probably not the best and could use some work. I'm good either way.